### PR TITLE
Minecraft cheat sheet: update controls

### PR DESCRIPTION
**Pull request by 122131**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`Minecraft cheat sheet: Controls update`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Updated minecraft controls according to http://minecraft.gamepedia.com/Controls

---

Instant Answer Page: https://duck.co/ia/view/minecraft_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @therealkbhat

### CODE REVIEW

**Comment by daxtheduck** [comment]
@122131 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/minecraft.json [hunk 1]
```diff
@@ -58,6 +58,10 @@
                 "key": "Q", 
                 "val": "Drop/toss item"
             },
+			{
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/minecraft.json]: Too much tabbing here.

File: share/goodie/cheat_sheets/json/minecraft.json [hunk 2]
```diff
@@ -121,11 +125,7 @@
             {
                 "key": "[D] [F3]", 
                 "val": "Clear chat history"
-            },
-            {
-                "key": "F8", 
-                "val": "Slowly pan camera"
-            }
+            },
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/minecraft.json]: Need to get rid of this comma since the last element in this list-block has been removed.

**Comment by 122131** [comment]
So I need to remove the comma, right?
Sorry, I'm new on this type of things...

**Comment by therealkbhat** [comment]
@122131 This looks good to me. Thanks for helping!

**Comment by gaulrobe** [comment]
@122131 yep, thanks for removing that pesky comma 😄 it made the JSON invalid.

**Comment by 122131** [comment]
It's been a pleasure!

**Comment by edgesince84** [comment]
@122131 thanks for the PR!  We'll get some more review for this ASAP!

**Comment by moollaza** [comment]
@122131 Thanks! This LGTM :+1:

**Pull request merged**